openff/toolkit/utils/toolkits.py

```diff
@@ -2114,6 +2115,86 @@ def generate_conformers(
         for conformer in molecule2._conformers:
             molecule._add_conformer(conformer)
 
+    def apply_elf_conformer_selection(
+        self,
+        molecule: "Molecule",
+        percentage: float = 2.0,
+        limit: int = 10,
+    ):
+        """Applies the `ELF method<https://docs.eyesopen.com/toolkits/python/quacpactk/
+        molchargetheory.html#elf-conformer-selection>`_ to select a set of diverse
+        conformers which have minimal electrostatically strongly interacting functional
+        groups from a molecules conformers.
+
+        Notes
+        -----
+        * The input molecule should have a large set of conformers already
+          generated to select the ELF conformers from.
+        * The selected conformers will be retained in the `molecule.conformers` list
+          while unselected conformers will be discarded.
+
+        See Also
+        --------
+        ``RDKitToolkitWrapper.apply_elf_conformer_selection``
+
+        Parameters
+        ----------
+        molecule
+            The molecule which contains the set of conformers to select from.
+        percentage
+            The percentage of conformers with the lowest electrostatic interaction
+            energies to greedily select from.
+        limit
+            The maximum number of conformers to select.
+        """
+
+        from openeye import oechem, oequacpac
+
+        if molecule.n_conformers == 0:
+            return
+
+        oe_molecule = molecule.to_openeye()
+
+        # Select a subset of the OMEGA generated conformers using the ELF10 method.
+        charge_engine = oequacpac.OEELFCharges(
+            oequacpac.OEAM1Charges(), limit, percentage, True
+        )
+
+        output_stream = oechem.oeosstream()
+
+        oechem.OEThrow.SetOutputStream(output_stream)
+        oechem.OEThrow.Clear()
+
+        status = oequacpac.OEAssignCharges(oe_molecule, charge_engine)
+
+        oechem.OEThrow.SetOutputStream(oechem.oeerr)
+
```

Comment:
(not blocking) Could you add (in a comment here) an example of what the error messages look like? I've found it's super helpful to have a raw text snippet when including parsing code like this, especially if the output format breaks some day and we don't know what input this was assuming.

Reply:
Hmm I'm not actually sure what error message will be raised here (especially with the new API point) - I'll play around a bit and see if I can get one to generate, but otherwise will pass on this until we get a real world example!